vm/vm.go

```diff
@@ -126,6 +127,8 @@ func Create(cfg *mgrconfig.Config, debug bool) (*Pool, error) {
 		template:   cfg.WorkdirTemplate,
 		timeouts:   cfg.Timeouts,
 		hostFuzzer: cfg.SysTarget.HostFuzzer,
+		statOutputReceived: stats.Create("vm output", "Bytes of VM console output received",
```

Comment:
Will several `stats.Create()` calls be a problem if we create the VM pool several times during execution (e.g. in syz-ci)?

Reply:
syz-ci also does not care about metrics, but it's probably better to merge metrics with the same name (?)
Prepended a commit that does it. Does it look reasonable?

Reply:
No, it broke lots of tests (e.g corpus stat that are now accumulated across tests).

Let's keep the current code. It's not crashing and until we understand what exactly behavior we want to get for syz-ci, it's hard to implement it.